railties/test/application/mailer_previews_test.rb

```diff
@@ -299,7 +300,7 @@ def foo
       app("development")
       get "/rails/mailers/notifier"
       assert_predicate last_response, :not_found?
-      assert_match "Mailer preview &#39;notifier&#39; not found", last_response.body
+      assert_match "Mailer preview &#39;notifier&#39; not found", h(last_response.body)
```

Comment:
Unfortunately without this change, this test was failing on rails 2.7. If I updated to `"Mailer preview 'notifier' not found"` this test failed on rails 2.6.5

Example of test failure without this change on 2.7

```
Failure:
ApplicationTests::MailerPreviewsTest#test_mailer_preview_not_found [test/application/mailer_previews_test.rb:303]:
Expected /Mailer\ preview\ &\#39;notifier&\#39;\ not\ found/ to match # encoding: ASCII-8BIT
```

Html from mailer (snipped)
```
 <h1>Unknown action</h1>\n</header>\n<div id=\"container\">\n    <div class=\"exception-message\">\n    <div class=\"message\">Mailer preview 'notifier' not found</div>\n  </div>\n\n</div>\n\n\n</body>\n</html>\n".
```